src/System.IO.FileSystem/tests/Directory/GetFiles_str_str_so.cs

```diff
@@ -8,642 +8,321 @@
-            try
+    [Fact]
+    public void Scenario2()
+    {
+        //Scenario 2: create a directory only top level files and directories and check
+        String dirName;
+        String[] expectedFiles;
+        String[] files;
+        List<String> list;
+
+        dirName = ManageFileSystem.GetNonExistingDir(Directory.GetCurrentDirectory(), ManageFileSystem.DirPrefixName);
+        using (ManageFileSystem fileManager = new ManageFileSystem(dirName, 1, 10))
+        {
+            expectedFiles = fileManager.GetAllFiles();
+            list = new List<String>(expectedFiles);
+            files = Directory.GetFiles(dirName, "*.*", SearchOption.AllDirectories);
+            Eval(files.Length, list.Count, "Err_202wur! wrong file count.");
+            for (int i = 0; i < files.Length; i++)
             {
-                dirName = ManageFileSystem.GetNonExistingDir(Directory.GetCurrentDirectory(), ManageFileSystem.DirPrefixName);
-                using (ManageFileSystem fileManager = new ManageFileSystem(dirName, 3, 100))
-                {
-                    expectedFiles = fileManager.GetFiles(dirName, 0);
-                    list = new List<String>(expectedFiles);
-                    files = Directory.GetFiles(dirName, "*.*", SearchOption.TopDirectoryOnly);
-                    Eval(files.Length, list.Count, "Err_763pjg! wrong file count.");
-                    for (int i = 0; i < files.Length; i++)
-                    {
-                        if (Eval(list.Contains(files[i]), "Err_512kvk! unexpected file found: {0}", files[i]))
-                            list.Remove(files[i]);
-                    }
-                    if (!Eval(list.Count == 0, "Err_839rbd! {0} expected files not found.", list.Count))
-                    {
-                        foreach (String fileName in list)
-                            Console.WriteLine(fileName);
-                    }
-                }
+                if (Eval(list.Contains(files[i]), "Err_611lgv! unexpected file found: {0}", files[i]))
+                    list.Remove(files[i]);
             }
-            catch (Exception ex)
+            if (!Eval(list.Count == 0, "Err_648ibm! {0} expected files not found.", list.Count))
             {
-                s_pass = false;
-                Console.WriteLine("Err_997pvx! Exception caught in scenario: {0}", ex);
+                foreach (String fileName in list)
+                    Console.WriteLine(fileName);
             }
+        }
+    }
 
-            // Path is not in the current directory (same drive)
-            //Scenario 4: Ensure that the path contains subdirectories and call this API and ensure that only the top directory files are returned
-            /* Scenario disabled when porting because it modifies the filesystem outside of the test's working directory
-            try
+    [Fact]
+    [ActiveIssue(0, PlatformID.Any)]
```

Comment:
Do we know why this was previously commented out?  If it's just a bad test, now would be a good time to remove it.  If it has value and needs fixing, it'd be good to fix it.  If that's too much work for now, we could tag it with a real issue number rather than 0.